### PR TITLE
Futher fixes for G3

### PR DESCRIPTION
**Pull request by tbosch**

### CODE REVIEW

File: packages/bazel/src/ng_module.bzl [hunk 1]
```diff
@@ -106,6 +106,7 @@ def _compile_action(ctx, inputs, outputs, config_file_path):
   if hasattr(ctx.attr, "tsconfig") and ctx.file.tsconfig:
     action_inputs += [ctx.file.tsconfig]
 
+  arguments = ["--node_options=--expose-gc"]
   # One at-sign makes this a params-file, enabling the worker strategy.
   # Two at-signs escapes the argument so it's passed through to ngc
   # rather than the contents getting expanded.
```
Comment (alexeagle) [inline packages/bazel/src/ng_module.bzl]: missing the `+=` below

File: packages/bazel/src/ngc-wrapped/index.ts [hunk 2]
```diff
@@ -47,49 +46,62 @@ function runOneBuild(args: string[], inputs?: {[path: string]: string}): boolean
...
+  // NB: the rootDirs should have been sorted longest-first
+  for (const dir of rootDirs || []) {
+    const rel = path.relative(dir, filePath);
+    if (rel.indexOf('.') != 0) return rel;
+  }
+  return filePath;
+}
+
+export function compile(
+    {fileLoader, compilerOpts, bazelOpts, files, expectedOuts, gatherDiagnostics}: {
```
Comment (alexeagle) [inline packages/bazel/src/ngc-wrapped/index.ts]: are you sure about taking named parameters here? so clunky

File: packages/compiler-cli/test/transformers/lower_expressions_spec.ts [hunk 3]
```diff
@@ -28,6 +28,11 @@ describe('Expression lowering', () => {
           class MyClass {}
       `)).toContain('const l = () => null; exports.l = l;');
     });
+
+    it('should be able to export a variable if the whole value is lowered', () => {
+      expect(convert('/*a*/ const a =◊b: () => null◊;'))
```
Comment (alexeagle) [inline packages/compiler-cli/test/transformers/lower_expressions_spec.ts]: weird diamond characters appear on github web ui :(

**Review by alexeagle — APPROVED** [review]

File: packages/compiler-cli/test/transformers/lower_expressions_spec.ts (hunk 3 above)
Comment (tbosch) [reply packages/compiler-cli/test/transformers/lower_expressions_spec.ts]: Yes, that is on purpose. See the other tests.

File: packages/bazel/src/ng_module.bzl (hunk 1 above)
Comment (tbosch) [reply packages/bazel/src/ng_module.bzl]: Done

File: packages/bazel/src/ngc-wrapped/index.ts (hunk 2 above)
Comment (tbosch) [reply packages/bazel/src/ngc-wrapped/index.ts]: Yes. This is way easier to read on the call site than to take all of these as positional arguments. We have it in the ngc api as well. And chuck also complained about it :-)

**Review by IgorMinar — APPROVED** [review]

**Comment by matsko** [comment]
Landed as a69172f6ce7553781ada03d733ca5a8f5774298d

**Pull request closed without merging**